Peer.GitHub/GraphQL/PullRequestSearch/CheckConclusionState.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace Peer.GitHub.GraphQL.PullRequestSearch
+{
+    public enum CheckConclusionState
+    {
+        Action_Required,
+        Cancelled,
+        Failure,
+        Neutral,
+        Skipped,
+        Stale,
+        Startup_Failure,
+        Success,
+        Timed_Out
+    }
+#nullable enable
```

Comment:
?

Reply:
Ah probably just the refactor -> new file and tagged along

Reply:
fixed